### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: Label for the modal dialog

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8912]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2516
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Label for the modal dialog

Issue Description:
There is no label for this modal dialog

Expected:
Provide appropriate label

User Impact:
If the dialog does not have an appropriate label, screen reader users find it difficult to understand its purpose.

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by gopinathan-n** [comment]
Screen reader is announcing the label now. Issue is fixed. Tested in Web components Lightbox media with carousel [https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/9149/index.html?path=/story/overview-getting-started--page](url)
